git-cliff/src/lib.rs

```diff
@@ -324,6 +306,42 @@ fn process_repository<'a>(
 	Ok(releases)
 }
 
+fn fill_release(
```

Comment:
Do we need separate functions for these? I feel like they are additional complexity.

I'm okay with it if we test them though.